ci/deploy.sh

```diff
@@ -59,7 +59,7 @@ fi
 
 # Commit message
 MESSAGE="\
-$(git log --max-count=1 --format='%s')
+$(git log --max-count=1 --format='%s') [ci skip]
```

Comment:
This has to be in the title of the commit message to work?

Reply:
Yes, AppVeyor requires it to be in the first line of the commit message.  Check my build history https://ci.appveyor.com/project/agitter/my-manuscript/history

Build `1.0.64` ran on the `output` branch commit https://github.com/agitter/my-manuscript/commit/08eef2c19a79f3f6c6ec7ced196d9e1d9b891264 even though it contains `[ci skip]` in the middle of the commit message.  It didn't run on https://github.com/agitter/my-manuscript/commit/dbadde6859c2b23c0758e3e3fde6ecfca4063c2a once I moved `[ci skip]` to the first line.

Reply:
> Yes, AppVeyor requires it to be in the first line of the commit message.

Looked into this a bit more. [They require it](https://help.appveyor.com/discussions/problems/543-skip-ci-not-working#comment_44000370) to be before any blank lines in the commit message. A misguided implementation IMHO.

Reply:
If AppVeyor looks for blank lines, should we change this to
```
$(git log --max-count=1 --format='%s')
[ci skip]
```

Reply:
We can do

```
$(git log --max-count=1 --format='%s')
[ci skip]
```

So that [ci skip] is on the second line. Not really standard git practice but perhaps better than lengthening the title.